docs/sources/next/extensions/explore.md

```diff
@@ -303,11 +259,7 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-stomp</h4>
         <p>Client for STOMP protocol</p>
     </a>
-    <a href="https://github.com/NAlexandrov/xk6-tcp" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements
https://github.com/NAlexandrov/xk6-tcp/issues/7